src/Integration/Binding/IProjectBinder.cs

```diff
@@ -29,7 +29,7 @@ namespace SonarLint.VisualStudio.Integration.Binding
     /// </summary>
     internal interface IProjectBinder
     {
-        bool IsBound(BindingConfiguration binding, Project project);
+        bool IsBindingRequired(BindingConfiguration binding, Project project);
```

Comment:
This deserves a doc comment to make it clear that the method will return `false` if the project isn't supported by the binder or if the project is already bound correctly i.e. it will only return `true` for unbound/partially-bound projects.